### ISSUE TITLE
WCAGへの参照がブレている

### ISSUE THREAD

**Issue by momdo** [bakera/html-book#191]
（Slackでも少し話をした記憶がありますが）

- https://github.com/bakera/html-book/blob/master/md_text/4-1.md#%E3%82%A2%E3%82%AF%E3%82%BB%E3%82%B7%E3%83%93%E3%83%AA%E3%83%86%E3%82%A3%E3%81%AE%E8%80%83%E6%85%AE
- https://github.com/bakera/html-book/blob/master/md_text/4-4.md#%E3%83%A1%E3%83%8B%E3%83%A5%E3%83%BC%E3%81%AE%E8%A3%8F%E5%81%B4%E3%81%AB%E3%83%95%E3%82%A9%E3%83%BC%E3%82%AB%E3%82%B9%E3%81%8C%E5%BD%93%E3%81%9F%E3%82%89%E3%81%AA%E3%81%84%E3%82%88%E3%81%86%E3%81%AB%E3%81%99%E3%82%8B

WCAG2だったり、WCAG 2.0だったり、WCAG 2.1だったり、
達成基準名も英語と日本語が混じっているので統一する必要あり。

**Comment by bakera** [comment]
問題がいくつかありますね。
まず、2.0を参照するのか2.1を参照するのかという問題です。
特に必要性がない場合は勧告になっている最新のもの、すなわち2.1を参照するのが良いように思います。

もう一つの問題は文書の参照時のルールです。文書の参照についての基本的な考え方は以下のようにしています。

- W3Cの文書を参照するときは原文を参照する (あくまで原文が正式版であるため)
- 英文の文書のタイトルやセクション名を表記するときは、英文のまま参照し、日本語には訳さない

ただし、これに従うとWCAGの参照は原文へリンクすることになり、達成基準の名前も英語で書くことになります。わかりにくい、長いという問題があり……。
WCAGだけWAICの翻訳を参照するという選択肢もあるのですが上記のルールとは食い違ってきます。

**Comment by momdo** [comment]
> まず、2.0を参照するのか2.1を参照するのかという問題です。
特に必要性がない場合は勧告になっている最新のもの、すなわち2.1を参照するのが良いように思います。

WCAG 2.1にあわせましょう。

> - W3Cの文書を参照するときは原文を参照する (あくまで原文が正式版であるため)
> - 英文の文書のタイトルやセクション名を表記するときは、英文のまま参照し、日本語には訳さない

はい、この基本方針の考え方は問題ないかと。

> ただし、これに従うとWCAGの参照は原文へリンクすることになり、達成基準の名前も英語で書くことになります。わかりにくい、長いという問題があり……。

https://github.com/bakera/html-book/issues/223#issuecomment-939393968 で案を出してみましたが、

- A案：達成基準名を記載せず、達成基準の番号だけを記載する
- B案：A案の変種で、達成基準名だけ日本語訳から借りてくる

A案であれば、基本方針には反しないかと。
B案は達成基準名を日本語訳で記載できる格好にはなります。

- C案：達成基準もURLもWAIC訳

パターンとしてはあり得ますが、WCAGの性質上、達成基準だけ読んでも理解できず、またWCAG 2.1解説書にリンクを張るわけでもないので、WAICにURLを飛ばすのは基本方針に反してまでしなくても良いのかなとは。

**Comment by bakera** [comment]
ありがとうございます。B案に1票です。
